Added noncentral-chisquared, noncentral-f and logseries distributions (#8520)
numba/numba: docs/source/reference/numpysupported.rst

* :func:`numpy.random.Generator().laplace()` (*)
* :func:`numpy.random.Generator().logistic()` (*)
* :func:`numpy.random.Generator().lognormal()` (*)
* :func:`numpy.random.Generator().logseries()` (Accepts float values as well as datatypes

>   that cast to floats. Array values for p are currently not supported.)

Reviewer: ```suggestion
  that cast to floats. Array values for ``p`` are currently not supported.)
```
and line wrap to 80 chars as needed.

* :func:`numpy.random.Generator().negative_binomial()` (*)

> * :func:`numpy.random.Generator().noncentral_chisquare()` (Accepts float values as well as datatypes

Reviewer: line wrap to 80 chars?

Author: Resolved in 280f269

Author: Resolved in 280f269

>   that cast to floats. Array values for dfnum and nonc are currently not supported.)

Reviewer: ```suggestion
  that cast to floats. Array values for ``dfnum`` and ``nonc`` are currently not supported.)
```
and line wrap to 80 chars as needed.

> * :func:`numpy.random.Generator().logseries()` (Accepts float values as well as datatypes
>  that cast to floats. Array values for p are currently not supported.)
> * :func:`numpy.random.Generator().negative_binomial()` (*)
> * :func:`numpy.random.Generator().noncentral_chisquare()` (Accepts float values as well as datatypes
>  that cast to floats. Array values for dfnum and nonc are currently not supported.)
> * :func:`numpy.random.Generator().noncentral_f()` (Accepts float values as well as datatypes
>  that cast to floats. Array values for dfnum, dfden and nonc are currently not supported.)

Reviewer: ```suggestion
* :func:`numpy.random.Generator().logseries()` (Accepts float values as well as datatypes
  that cast to floats. Array values for p are currently not supported.)
* :func:`numpy.random.Generator().negative_binomial()` (*)
* :func:`numpy.random.Generator().noncentral_chisquare()` (Accepts float values as well as datatypes
  that cast to floats. Array values for dfnum and nonc are currently not supported.)
* :func:`numpy.random.Generator().noncentral_f()` (Accepts float values as well as datatypes
  that cast to floats. Array values for dfnum, dfden and nonc are currently not supported.)
```
I think something like this ought to fix the doc build. IIRC continued item lines in bulleted lists need the same indent level as the start of the item.

>   that cast to floats. Array values for dfnum, dfden and nonc are currently not supported.)

Reviewer: ```suggestion
  that cast to floats. Array values for ``dfnum``, ``dfden`` and ``nonc`` are currently not supported.)
```
and line wrap to 80 chars as needed.

* :func:`numpy.random.Generator().laplace()` (*)
* :func:`numpy.random.Generator().logistic()` (*)
* :func:`numpy.random.Generator().lognormal()` (*)
* :func:`numpy.random.Generator().logseries()` (Accepts float values as well as

>   datatypes that cast to floats. Array values for ``p`` are currently not

Reviewer: ```suggestion
  data types that cast to floats. Array values for ``p`` are currently not
```

  supported.)
* :func:`numpy.random.Generator().negative_binomial()` (*)
* :func:`numpy.random.Generator().noncentral_chisquare()` (Accepts float values

>   as well as datatypes that cast to floats. Array values for ``dfnum`` and

Reviewer: ```suggestion
  as well as data types that cast to floats. Array values for ``dfnum`` and
```

  ``nonc`` are currently not supported.)
* :func:`numpy.random.Generator().noncentral_f()` (Accepts float values as well

>   as datatypes that cast to floats. Array values for ``dfnum``, ``dfden`` and

Reviewer: ```suggestion
  as data types that cast to floats. Array values for ``dfnum``, ``dfden`` and
```
